src/Elmish.WPF/ViewModel.fs

```diff
@@ -95,7 +98,9 @@ and [<AllowNullLiteral>] internal ViewModel<'model, 'msg>
     sprintf "%s.%s.%s" propNameChain collectionBindingName itemId
 
   let notifyPropertyChanged propName =
-    log "[%s] PropertyChanged \"%s\"" propNameChain propName
+    logTrace <| PropertyChangedData {
```

Comment:
We should definitely make this lazy going forward. Only instantiate stuff if logging is actually enabled.